src/data/text/move_descriptions.h

```diff
@@ -2766,10 +2818,97 @@ static const u8 sSTEEL_BEAMDescription[] = _(
     "Fires a beam of steel from\n"
     "its body. It hurts the user.");
 
+static const u8 sEXPANDING_FORCEDescription[] = _(
+    "A Psychic attack that's more\n"
+    "powerful in Psychic Terrain.");
+
+static const u8 sSTEEL_ROLLERDescription[] = _(
+    "Destroys the current terrain."
+    "Otherwise, it fails.");
+
+static const u8 sSCALE_SHOTDescription[] = _(
+    "Shoots scales 2 to 5 times.\n"
+    "Ups Speed, lowers defense.");
+
+static const u8 sMETEOR_BEAMDescription[] = _(
+    "A 2-turn move that raises\n"
+    "Sp. Attack before attacking.");
+
+static const u8 sSHELL_SIDE_ARMDescription[] = _(
+    "Its category depends on\n"
+    "which one will damage more.");
+
+static const u8 sMISTY_EXPLOSIONDescription[] = _(
+    "Boosted by Misty Terrain, it\n"
+    "makes the user faint.");
+
+static const u8 sGRASSY_GLIDEDescription[] = _(
+    "On Grassy Terrain, the user\n"
+    "will always strike first.");
+
+static const u8 sRISING_VOLTAGEDescription[] = _(
+    "Its power doubles while\n"
+    "on Electric Terrain.");
+
+static const u8 sTERRAIN_PULSEDescription[] = _(
+    "Type changes depending on\n"
+    "terrain. Power also doubles.");
+
+static const u8 sSKITTER_SMACKDescription[] = _(
+    "User skitters behind target\n"
+    "to attack. Lowers Sp. Atk.");
+
+static const u8 sBURNING_JEALOUSYDescription[] = _(
+    "Burns whoever their stats\n"
+    "raised on the same turn.");
+
+static const u8 sLASH_OUTDescription[] = _(
+    "Power doubles if stats were\n"
+    "lowered on the same turn.");
+
+static const u8 sPOLTERGEISTDescription[] = _(
+    "Only works if the target\n"
+    "is holding an item.");
+
+static const u8 sCORROSIVE_GASDescription[] = _(
+    "Melts all surrounding items\n"
+    "with highly acidic gas.");
+
+static const u8 sCOACHINGDescription[] = _(
+    "The user coaches an ally to\n"
+    "raise their Atk. and Def.");
+
+static const u8 sFLIP_TURNDescription[] = _(
+    "Does damage then switches\n"
+    "out the user.");
+
+static const u8 sTRIPLE_AXELDescription[] = _(
+    "A 3-kick attack that gets\n"
+    "stronger on each hit.");
+
+static const u8 sDUAL_WINGBEATDescription[] = _(
+    "The user uses its wings to\n"
+    "slam the target twice.");
+
+static const u8 sSCORCHING_SANDSDescription[] = _(
+    "It throws scorching sand\n"
+    "that may burn the target.");
+
+static const u8 sJUNGLE_HEALINGDescription[] = _(
+    "Becomes one with the jungle\n"
+    "to heal the team's health.");
+
+static const u8 sWICKED_BLOWDescription[] = _(
+    "By mastering the Dark style,\n"
+    "always does a critical hit.");
+
+static const u8 sSURGING_STRIKESDescription[] = _(
+    "The Water style allows to\n"
```

Comment:
"allows to do 3 critical hits" is awkward phrasing imo. I'd probably do "strikes with 3 critical hits" or "lands with 3 critical hits"